TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/objects/Server.java

```diff
@@ -416,7 +416,7 @@ public ServerObject toServerObject() {
                 getExtra(),
                 getMap(),
                 getMotd(),
-                getOnlinePlayers().stream().map(player -> ((PlayerObjectBasicImplementation) player).toLink()).collect(Collectors.toSet()),
+                new HashSet<>(getOnlinePlayers()).stream().map(player -> ((PlayerObjectBasicImplementation) player).toLink()).collect(Collectors.toSet()),
```

Comment:
Why are you converting the online players list to a `HashSet` before streaming it?